analysis/topEFT/remake_ci_ref_yields.sh

```diff
@@ -3,7 +3,7 @@
 
 # Get the file the CI uses, and move it to the directory the JSON expects
 printf "\nDownloading root file...\n"
-wget -nc http://www.crc.nd.edu/~ywan2/root_files/NAOD-00000_18449.root ../../NAOD-00000_18449.root
+wget http://www.crc.nd.edu/~kmohrman/files/root_files/for_ci/ttHJet_UL17_R1B14_NAOD-00000_10194.root
```

Comment:
We should keep the `-nc` flag to prevent redownloading if the file already exists locally.

Reply:
Thanks for catching that @bryates, I guess I accidentally removed that flag when pasting in the path to the new file. I've added the `-nc` back in now.

Reply:
No worries, thanks for the fix.